packages/scaffolding-cli/templates/src/ssr/environment-configuration/environment-variables.js

```diff
@@ -13,4 +13,14 @@ module.exports = {
     AUTH0_DOMAIN: process.env.AUTH0_DOMAIN,
     AUTH0_CLIENT_SECRET: process.env.AUTH0_CLIENT_SECRET,
     AUTH0_CALLBACK_URL: `${process.env.APP_BASE_URL}:${process.env.PORT}/callback`,
+
+    NEXT_PUBLIC_CONTENTFUL_SPACE_ID:
+        process.env.NEXT_PUBLIC_CONTENTFUL_SPACE_ID || "veka2or2977j",
```

Comment:
Is this default a public space id?

Reply:
I left it default which points to the our stacks space.